### ISSUE TITLE
Zettels with title ID, but no header, are unavailable in `neuron search`

### ISSUE THREAD

**Issue by srid** [srid/neuron#439]
The [search script](https://github.com/srid/neuron/blob/master/neuron/src-bash/neuron-search) should be adapted to deal with implicit title in the filename, when one is not provided in the note body. 

To reproduce:

```
echo Test > "Foo bar.md"
neuron search
```

"Foo bar" won't appear in search.

**Comment by srid** [comment]
Either extend the `neuron-search` script (like #445), or rewrite the bulk of it in Haskell.

**Comment by LATBauerdick** [comment]
this works now with the merge of [445](https://github.com/srid/neuron/pull/445). Could be closed, although the Haskell rewrite is still desirable.

**Comment by srid** [comment]
@LATBauerdick This is *almost* fixed by #445, but it doesn't handle an edge case, wherein *all* of your notes use title ID.

To repro:

```
mkdir /tmp/repro
cd /tmp/repro/ 
echo Test > "Foo bar.md"
echo Test2 > "Qux.md"
neuron search
```

**Comment by LATBauerdick** [comment]
hmm, embarrassing mistake... need to replace the `&&` in `neuron/search-bash/` with a `;`, that's it...
